consensus/common_test.go

```diff
@@ -266,17 +285,26 @@ func decideProposal(
 		panic("Failed to createProposalBlock. Did you forget to add commit for previous block?")
 	}
 
+	var (
+		blob      = types.Blob(cmtrand.Bytes(42))
```

Comment:
In line 276 the original PR (L285 in that case) is calling the newly created CreateProposalBlockAndBlob function. You are here creating a proposal w/o a blob. If you call that function you will not need to manually create a blob here.

Reply:
Same, the relevant commit was not cherry-picked at the time of review. Now it should be fine.